Examples/DataRepresentation/Mesh/MeshCellVisitor.cxx

```diff
@@ -84,8 +84,8 @@ using TetrahedronType = itk::TetrahedronCell<CellType>;
 //  Software Guide : EndLatex
 
 
-#ifndef __CustomTriangleVisitor
-#  define __CustomTriangleVisitor
+#ifndef CUSTOM_TRIANGLE_VISITOR
+#  define CUSTOM_TRIANGLE_VISITOR
```

Comment:
Isn't ITK's style CamelCase?
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkArray.h#L18
Or does that apply only for include guards?